lambda.tf

```diff
@@ -67,12 +67,14 @@ data "archive_file" "lambda" {
 }
 
 module "lambda" {
+  #checkov:skip=CKV_AWS_272: This module does not provide support for code-signing
   providers        = { aws.lambda = aws.lambda }
   source           = "github.com/schubergphilis/terraform-aws-mcaf-lambda?ref=v0.3.6"
   name             = var.lambda_name
   description      = "Forwards email sent to recipients in the \"${var.ses_rule_set_name}\" SES Rule Set to external addresses"
   filename         = data.archive_file.lambda.output_path
   handler          = "index.handler"
+  kms_key_arn      = var.kms_key_arn
```

Comment:
In this case its not adding much of security, but right now this module cannot be used if you have a very strict KMS key policy. E.g. we have a policy that checks that all lambda's have a KMS key policy set in one of our customer teams. They are not able to use this module right now.

Reply:
Ok, now I understand why we need it but this may cause a side effect. See this from the documentation -
If this configuration is provided when environment variables are not in use, the AWS Lambda API does not save this configuration and Terraform will show a perpetual difference of adding the key. To fix the perpetual difference, remove this configuration.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function

Reply:
Thank you for the sharp eye, this is indeed not desired, i'll close this PR and will solve this in another way